roles/optional-operators-subscribe/templates/optional-operators-subscribe-commands.sh

```diff
@@ -205,4 +207,5 @@ else
     echo "Dumping all ClusterServiceVersions in namespace $OO_INSTALL_NAMESPACE to $CSV_ART"
     oc get -n "$OO_INSTALL_NAMESPACE" csv -o yaml >"$CSV_ART"
 fi
+printf '{"timestamp":"%s","passed": false }' "${DEPLOYMENT_START_TIME}" > "${ARTIFACT_DIR}/finished.json"
```

Comment:
Actually, I have to rescind my LGTM.  Because this script is shared with DPTP (we just copy the file from `openshift/release`), we can't make changes here without submitting them in that repo as well.  Can you move the generation of this file to the ansible playbook. The `collect_modified_images` role has an example of how to get the timestamps if you don't know how.

Reply:
Got it. Thanks

Reply:
@14rcole How specific are you about the timestamp. 
Would the timestamp mean about the deployment start time ( which is a dptp delpoyment start time am not sure about how to get it ) ? 
the optional operator subscribe step start time ? 
If the later  i can add a variable to fetch the timestamp within this playbook ...
could you clarify

Reply:
If by the second you mean the start time for the shell script then yes, do that

The script itself is the deployment.  When this playbook is run, we won't actually be calling out to DPTP

Reply:
I have made necessary changes please re-review 
@14rcole